starport/pkg/gomodulepath/gomodulepath.go

```diff
@@ -48,8 +48,8 @@ func Parse(rawpath string) (Path, error) {
 	return p, nil
 }
 
-// ParseFile parses Go module path of an app resides at path.
-func ParseFile(path string) (Path, error) {
+// ParseAt parses Go module path of an app resides at path.
```

Comment:
It was already here but I don't understand the function description

Reply:
`ParseFile` was more like parsing a file but what we do here is to parse the relevant file that located under the path so, I thought `ParseAt` is more suitable.